concent_api/verifier/utils.py

```diff
@@ -354,3 +354,61 @@ def download_archives_from_storage(file_transfer_token, subtask_id, package_path
                 ErrorCode.VERIFIER_FILE_DOWNLOAD_FAILED,
                 subtask_id=subtask_id,
             )
+
+
+def parse_result_files_with_frames(frames, result_files_list, output_format):
+    frames_to_result_files_map = {}
+    for frame_number in frames:
+        for result_file_name in result_files_list:
+            if re.search(f'_{frame_number:>04}.{output_format.lower()}$', result_file_name) is not None:
+                if result_file_name not in frames_to_result_files_map.values():
+                    frames_to_result_files_map[frame_number] = [generate_verifier_storage_file_path(result_file_name)]
+    return frames_to_result_files_map
+
+
+def render_images_by_frames(parsed_files_to_compare, frames, output_format, scene_file, subtask_id):
```

Comment:
You can add typings and a comment to each of the functions below.

Reply:
Done